libportability-gfx/src/impls.rs

```diff
@@ -2358,7 +2358,8 @@ pub extern "C" fn gfxCreateDescriptorPool(
     let pool = super::DescriptorPool {
         raw: gpu.device
             .create_descriptor_pool(info.maxSets as _, ranges),
-        sets: if info.flags & VkDescriptorPoolCreateFlagBits::VK_DESCRIPTOR_POOL_CREATE_FREE_DESCRIPTOR_SET_BIT as u32 != 0 {
+        temp_sets: Vec::new(),
```

Comment:
we could actually use the `info.maxSets` for reserving memory here

Reply:
good idea! fixed in https://github.com/gfx-rs/portability/pull/117